docs/staging/AA/aa-new-review.md

```diff
@@ -320,4 +422,83 @@ Sometimes we might need to discuss with the uploader instead of just rejecting.
 In that case have a look at the `.dsc` file with `gpg –verify` to get the
 sponsor and the changelog to get the packager. Use that info to connect to them.
 
+## Accepting a package
+
+
+If the package is fine, you should next check that it's going to end up in the
+right part of the archive. On the next line of the `info` output, you have
+details about the different parts of the package, including which component,
+section, etc. it is expected to head into. One of the important jobs is making
+sure that this information is actually correct through the application of
+overrides.
+
+To alter the overrides for a package, use:
+
+```bash
+$ ./queue override -c universe ubuntustudio-menu
+```
+
+Where the override can be `-c <component>`, `-x <section>`, and/or (for binary
+packages) `-p <priority>`. If you want to limit the override application to
+only source or only binary packages, use the `--source` or `--binary` options
+respectively.
+
+Often, a binary will be in the `NEW` queue because it is a shared library that
+has changed `SONAME`. In this case you'll probably want to check the existing
+overrides to make sure anything new matches. These can be found in
+`/ubuntu/indices` on Ubuntu mirrors.
+
+Currently, a special case of this are the kernel packages, which change package
+names with each ABI update and build many distinct binary packages in different
+sections. A helper tool has been written to apply overrides to the queue based
+on the packages that are currently published:
+
+```bash
+$ ./kernel-overrides [-s <sourcepackage>] <newabi>
+```
+
+Binary packages are not often rejected (they go into a black hole with no
+automatic notifications), so, check the `.deb` contains files, run `lintian` on
+it and file bugs when things are broken. The binaries also need to be put into
+`universe` etc as appropriate even if the source is already there.
```

Comment:
The meaning of this para is not clear to me... Suggesting a change but need an ack (@cpaelzer, please?) that it's the actual intended meaning.
```suggestion
Binary packages are not often rejected (they are just discarded without any
automatic notifications). So, check that the `.deb` contains files, run `lintian` on
it, and file bugs when things are broken. The binaries also need to be put into
`universe` or other repositories as appropriate even if the source is already there.
```